pkgs/applications/networking/cluster/cmctl/default.nix

```diff
@@ -19,6 +24,8 @@ buildGoModule rec {
     "-s" "-w"
     "-X github.com/cert-manager/cert-manager/cmd/ctl/pkg/build.name=cmctl"
     "-X github.com/cert-manager/cert-manager/cmd/ctl/pkg/build/commands.registerCompletion=true"
+    "-X github.com/cert-manager/cert-manager/pkg/util.AppVersion=v${version}"
+    "-X github.com/cert-manager/cert-manager/pkg/util.AppGitCommit=${src.rev}"
```

Comment:
```suggestion
    "-X github.com/cert-manager/cert-manager/pkg/util.AppVersion=v${src.rev}"
    "-X github.com/cert-manager/cert-manager/pkg/util.AppGitCommit=${src.rev}"
```

Reply:
Unless I’m mistaken, the problem here is that `AppGitCommit` is no longer going to be the commit

Reply:
Yeah, thats fine. Its not really worth it to mirror upstream one to one just for informational purposes.

Reply:
I disagree, I'd like to have the

Reply:
@SuperSandro2000 Update script is working for me. Only GITHUB_TOKEN is missing.